Sources/BraintreeThreeDSecure/Models/BTThreeDSecureAuthenticateJWTPOSTBody.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// The POST body for /three_d_secure/authenticate_from_jwt
```

Comment:
```suggestion
// The POST body for `/three_d_secure/authenticate_from_jwt`
```

Reply:
addressed in ff7119eae19aaf332abc9907f98ad1d3cb83ac9d